lib/Monads/reader_option/Reader_Option_Monad.thy

```diff
@@ -614,4 +619,49 @@ proof -
   then show ?thesis by (auto intro: final split: option.split_asm)
 qed
 
+
+\<comment> \<open>FIXME: better name for this section\<close>
+section \<open>Extra definitions\<close>
+
+\<comment> \<open>FIXME: The only one of these currently used is odrop, in sep-capDL/Lookups_D.
+          As far as I can tell, the definitions in Lookups_D aren't used. Should we
+          remove them, and these?\<close>
```

Comment:
I'm very much up for deleting them, yes!

Reply:
I might have been a bit too hasty with what I said here. I just checked again and parts of Lookups_D are still used in `capDL-api`, but the specific definitions that use these reader_option definitions aren't.

See https://github.com/seL4/l4v/commit/e75b38713bdcefa9dea0e77dbd341433c0f33e86 for what could be removed.

Reply:
That's still pretty good, let's do that.